### PR TITLE
Fix HTTP_URI typo in http.py

### LINKED ISSUE #140: Invalid HTTP URI format in http.py
https://github.com/Thriftpy/thriftpy2/blob/a8b7873078a6f38a2f052964d439ab888e32fcf4/thriftpy2/http.py#L63

It's missing the `/`  between the port and the path and should be:

```python
HTTP_URI = '{scheme}://{host}:{port}/{path}'
```

This only occurs if you leave out the `/` in your path variable.

### PR DESCRIPTION
**Pull request by aisk**
close #140

### CODE REVIEW

**Review by ethe — APPROVED** [review]

File: thriftpy2/http.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
 from thriftpy2.transport import TBufferedTransportFactory
 
 
-HTTP_URI = '{scheme}://{host}:{port}{path}'
+HTTP_URI = '{scheme}://{host}:{port}/{path}'
```
Comment (dmulter) [inline thriftpy2/http.py:63]: This change just broke things for me.

**Comment by aisk** [comment]
Really sorry for the break, so should we add a if statement and and the `/` if it not exists?

**Comment by dmulter** [comment]
Your call, but since the standard in `urlparse` is to have a starting slash, I would think it's the responsibility of the caller to make sure the path is in that format. Could also be clarified in the docs. Anyway, that would be my opinion.

**Comment by aisk** [comment]
@dmulter I think you are right, and really sorry for I did not make it right. Just open a new PR: https://github.com/Thriftpy/thriftpy2/pull/158 , can you help me review it?

**Pull request merged**